.zuul.yaml

```
@@ -4,6 +4,20 @@
     check:
       jobs:
         - noop
+        - dashboard-pre-commit
```

Shouldn't it replace the `noop` job?
```suggestion
        - dashboard-pre-commit
```